docs/reference/settings/notification-settings.asciidoc

```diff
@@ -286,19 +328,23 @@ issues in Jira. You can specify the following Jira account attributes:
 --
 [[jira-account-attributes]]
 
-  `secure_url` (<<secure-settings,Secure>>);;
-  The URL of the Jira Software server. Required.
+`secure_url`::
+(<<secure-settings,Secure>>)
+The URL of the Jira Software server. Required.
 
-  `secure_user` (<<secure-settings,Secure>>);;
-  The name of the user to connect to the Jira Software server. Required.
+`secure_user`::
+(<<secure-settings,Secure>>)
+The name of the user to connect to the Jira Software server. Required.
 
-  `secure_password` (<<secure-settings,Secure>>);;
-  The password of the user to connect to the Jira Software server. Required.
+`secure_password`::
+(<<secure-settings,Secure>>)
+The password of the user to connect to the Jira Software server. Required.
 
-  `issue_defaults`;;
-  Default fields values for the issue created in Jira. See
-  <<jira-action-attributes>> for more information.
-  Optional.
+`issue_defaults`::
+(<<static-cluster-setting,Static>>)
```

Comment:
Similar to the message_defaults setting, this affix setting seems to be configured as dynamic.